config/routes.rb

```diff
@@ -1,6 +1,11 @@
 Rails.application.routes.draw do
   root 'homes#index'
   devise_for :users
+
+  resources :beaches do
+    resources :reviews
+  end
```

Comment:
We decided to wait until we do update and destroy before dealing with limiting resources